### PR TITLE
Remove V1 wrapper: Timestamp.

### PR DESCRIPTION
**Pull request by denisshevchenko**
# Linked Issue

<Put here a reference to the issue this PR relates to and which requirements it tackles>

<p align="right">#119</p>

# Overview

<Detail in a few bullet points the work accomplished in this PR>

1. - [x] No `V1` wrapper for `Core.Timestamp` type.
2. - [x] `WalletTimestamp` newtype is using instead.
3. - [x] golden test.

# Comments

<Additional comments or screenshots to attach if any>

¯\\\_(ツ)\_/¯

# Checklist

- [x] I've kept this PR simple, on-the-spot, free of cosmetic changes.
- [x] I have reviewed my commit messages and history.
- [x] I've assigned a reviewer.
- [x] I acknowledge my changes may require an update in the wiki.
- [x] I have added a reference to this PR to the corresponding issue.

---

- [x] I've checked the checklist

### CODE REVIEW

File: test/unit/Golden/golden/apiV1Types/json/Timestamp [hunk 1]
```diff
@@ -0,0 +1 @@
+"2018-12-11T08:09:10"
```
Comment (KtorZ) [inline test/unit/Golden/golden/apiV1Types/json/Timestamp:1]: I assume you've generated this BEFORE making the change with V1 :stuck_out_tongue:  ? Right ?

Comment (KtorZ) [reply test/unit/Golden/golden/apiV1Types/json/Timestamp:1]: (would be nice if the commit history reflect this :wink:)

Comment (denisshevchenko) [reply test/unit/Golden/golden/apiV1Types/json/Timestamp:1]: Right. I checked it with `(V1 exampleTimestamp)`, and after that - with `(WalletTimestamp exampleTimestamp)`.

Comment (denisshevchenko) [reply test/unit/Golden/golden/apiV1Types/json/Timestamp:1]: > (would be nice if the commit history reflect this wink)

Sorry, I rebased (squash) it... :-)

**Review by KtorZ — APPROVED** [review]
LGTM :+1:

**Pull request merged**